src/java.base/share/classes/java/lang/classfile/attribute/ModuleResolutionAttribute.java

```diff
@@ -60,22 +61,26 @@
  * The attribute does not permit multiple instances in a given location.
  * Subsequent occurrence of the attribute takes precedence during the attributed
  * element build or transformation.
+ *
+ * @since 22
  */
+@PreviewFeature(feature = PreviewFeature.Feature.CLASSFILE_API)
 public sealed interface ModuleResolutionAttribute
         extends Attribute<ModuleResolutionAttribute>, ClassElement
         permits BoundAttribute.BoundModuleResolutionAttribute, UnboundAttribute.UnboundModuleResolutionAttribute {
 
     /**
      *  The value of the resolution_flags item is a mask of flags used to denote
      *  properties of module resolution. The flags are as follows:
-     *
+     * <pre> {@code
```

Comment:
Why not a snippet?

Reply:
It is a citation from the spec, not sure a snippet would be appropriate.